### PR TITLE
Add cogito, a Concourse GitHub status resource

### PR DESCRIPTION
**Pull request by marco-m**
Signed-off-by: Marco Molteni <marco.molteni@mailbox.org>

### CODE REVIEW

File: pix4d-cogito.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: cogito
+repo: https://github.com/Pix4D/cogito
+container_image: pix4d/cogito
+description: |
+Cogito updates the GitHub status of a commit during a build. Simple configuration and lightweight image.
```
Comment (jomsie) [inline pix4d-cogito.yml]: Hi! Just need some indentation here for valid yml!

```diff
-Cogito updates the GitHub status of a commit during a build. Simple configuration and lightweight image.
+  Cogito updates the GitHub status of a commit during a build. Simple configuration and lightweight image.
```

Thanks!

**Review by jomsie — CHANGES_REQUESTED** [review]
Thanks for the PR! Just one small change.

File: pix4d-cogito.yml (hunk 1 above)
Comment (marco-m) [reply pix4d-cogito.yml]: it's funny because I use so much YAML daily and didn't spot this one :-D

Comment (marco-m) [reply pix4d-cogito.yml]: Fixed.

Comment (YoussB) [reply pix4d-cogito.yml]: Thanks @marco-m

**Pull request merged**